README.md

```diff
@@ -354,6 +354,35 @@ for details.
     // remove the contact from the device
     contact.remove(onSuccess,onError);
 
+### Removing phone number(s) from a saved contact
+
+    var myContact = navigator.contacts.create({"displayName": "Test User"});
+    var phoneNumbers = [];
+
+    phoneNumbers[0] = new ContactField('work', '768-555-1234', false);
+    phoneNumbers[1] = new ContactField('mobile', '999-555-5432', true); // preferred number
+    phoneNumbers[2] = new ContactField('home', '203-555-7890', false);
+
+    myContact.phoneNumbers = phoneNumbers;
+    myContact.save(function (contact_obj) {
```

Comment:
The purpose is to provide a sample snippet. Adding code for find will just make it bigger (I already feel it is bigger and could be small) So, I am leaving it as such...